src/Microsoft.HttpRepl.IntegrationTests/Mocks/MockedShellState.cs

```diff
@@ -11,7 +11,7 @@
 
 namespace Microsoft.HttpRepl.IntegrationTests.Mocks
 {
-    internal class MockedShellState : IShellState
+    public class MockedShellState : IShellState
```

Comment:
@tlmii , had to update this due to inconsistent accessibility issue. This is one of the parameters in ArrangeInputs(..) in CommandTestsBase and the method is protected. Let me know if you have concerns/suggestions.